deploy/provider-gcp-plugin.yaml.tmpl

```diff
@@ -87,6 +90,14 @@ spec:
             - name: mountpoint-dir
               mountPath: /var/lib/kubelet/pods
               mountPropagation: HostToContainer
+          livenessProbe:
```

Comment:
I think that's mostly used for whether to route network requests to the pod

> Container will be removed from service endpoints if the probe fails [1](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#container-v1-core)

Since we don't use the Service Object I don't think its necessary

Reply:
ah then should we remove the /ready endpoint since it's unused?

Reply:
done!